### PR TITLE
Update protobuf to 0.52 GA version

### PR DESCRIPTION
**Pull request by edwin-greene**
**Description**:
Updates the protobuf version to the GA of 0.52.0

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by jascks — APPROVED** [review]
LGTM

File: build.gradle.kts [hunk 1]
```diff
@@ -66,7 +66,7 @@ dependencies {
         api("com.graphql-java:graphql-java-extended-scalars:22.0")
         api("com.graphql-java:graphql-java-extended-validation:22.0")
         api("com.hedera.evm:hedera-evm:0.48.0")
-        api("com.hedera.hashgraph:hedera-protobuf-java-api:0.52.0-token-reject-SNAPSHOT")
+        api("com.hedera.hashgraph:hedera-protobuf-java-api:0.52.0")
```
Comment (steven-sheehy) [inline build.gradle.kts:69]: Did you confirm there were no changes to the proto files like renamed or added response codes?

Comment (edwin-greene) [reply build.gradle.kts:69]: Thanks for mentioning that. I hadn't thought of it. There were 2 additions, I have added them to `transactionResult.js`

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by jascks — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**